src/saveload/engine_sl.cpp

```diff
@@ -189,8 +189,23 @@ struct EIDSChunkHandler : ChunkHandler {
 	{
 		SlTableHeader(_engine_id_mapping_desc);
 
+		/* Get total entries required for flat map. */
+		size_t total = 0;
+		for (const auto &mapping : _engine_mngr.mappings) { total += std::size(mapping); }
```

Comment:
Doesn't coding style practically dictate using newlines when using curly brackets for a `for`? Same with the next loop.